channels/tr.m3u

```diff
@@ -159,3 +159,115 @@ https://turkmedya-live.ercdn.net/tv4/tv4_720p.m3u8
 https://ucankus-live.cdnnew.com/ucankus/ucankus.m3u8
 #EXTINF:-1 tvg-id="VTV.tr" tvg-name="VTV" tvg-country="TR" tvg-language="Turkish" tvg-logo="https://i.imgur.com/FRripYx.png" group-title="",VTV
 https://waw2.artiyerelmedya.net/kanalv/bant1/chunks.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",1NTVSPOR HD
+http://46.4.193.238:8484/hls/ntvspor/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",24 HABER HD+
+http://163.172.39.215:25461/live/line/C4@!a3a1@!w72A/134.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",24 TV
+https://mn-nl.mncdn.com/kanal24/smil:kanal24.smil/chunklist_b896000.m3u8?__mn_sid=8997673748844384653
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",360
+http://mfe.cliptv.az/dash/360_TV_SD.ism/playlist.mpd
```

Comment:
This is Russian channel: https://www.lyngsat.com/tvchannels/ru/Telekanal-360-.html